core/mono-samples/iOS/README.md

```diff
@@ -0,0 +1,25 @@
+---
+languages:
+- c#
+products:
+- dotnet
+- iOS
+- mono runtime
+page_type: sample
+name: "iOS Sample: Simple greeting and counter (C#)"
+description: "An iOS application that contains an example of embedding the mono runtime to invoke unmanaged code with C#."
+---
+
+# iOS Sample: Simple greeting and counter (C#)
+
+In this sample, the mono runtime is used to invoke objective-c unmanaged code (main.m) from the C# managed side (Program.cs) and vice versa. With the sample running, you can enter your name and click the corresponding button to modify the greeting message as well as clicking a button to increment a counter.
+
+NOTE: The purpose of this sample is to demonstrate the concept of building an iOS application on top of the mono runtime.
+
+## Requirements
+
+To run the sample for iOS you will need a recent version of XCode installed (e.g. 11.3 or higher).
```

Comment:
This could be enhanced. Which versions of iOS? What version of Mono? Is there a link to XCode you could add? etc.

Reply:
I received some clarifications on the minimum versions necessary and added a link to downloading Xcode.